src/components/Sum.jsx

```diff
@@ -1 +1,5 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = ({ a = 0, b = 0 }) => (
+  <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
```

Comment:
not critical, but maybe it's better to format it like this: 
```suggestion
  <p>
    {`Sum of ${a} and ${b} is ${a + b}`}
  </p>
```